modules/copyrights.php

```diff
@@ -57,6 +57,12 @@
             'realname' => 'Krzysztof Drewicz',
             ),
 
+	'interduo' => array(
+            'info' => 'Frontend, Backend, Testowanie, Optymalizacja',
+	    'realname' => 'Jarosław Kłopotek',
+	    'www' => 'www.interduo.pl'
+            ),
+
         'Lexx' => array(
```

Comment:
No - o to już dawno prosiłem tylko dlaczego przemycone w taki sposób, że nie da się prosto zrobić cherry-pick tego co stable i stable-24?

Reply:
Załóżmy że dopiero teraz zasłużyłem żeby być w zacnym gronie TOP3.

Reply:
Nie no są dwa commity zrobiłem rebase wcześniej. Nie wiem ocb